### PR TITLE
Add Social Links of Medium and GitHub

### PR DESCRIPTION
**Pull request by sharonibose**
## Related Issue
Fixed Issue #1145.

## Proposed Changes
Added social links of Medium and GitHub.

## Checklist

- [x] My PR follows the style guidelines of this project.
- [x] I have made corresponding changes to the documentation.
- [x] Any dependent changes have been merged.

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -51,6 +51,8 @@ module.exports = {
             <a href="https://twitter.com/anitab_org" rel="noopener noreferrer" target="_blank"><i id="social-tw" class="fa fa-twitter-square fa-3x social"></i></a>
             <a href="https://www.linkedin.com/company/anitab-org/" rel="noopener noreferrer" target="_blank"><i id="fa fa-linkedin-square fa-3x social" class="fa fa-linkedin-square fa-3x social"></i></a>
             <a href="https://www.instagram.com/anitab_org/" rel="noopener noreferrer" target="_blank"><i id="fa fa-instagram-square fa-2x social" class="fa fa-instagram fa-3x social"></i></a>
+            <a href="https://medium.com/anitab-org-open-source" rel="noopener noreferrer" target="_blank"><i id="fa fa-medium" class="fa fa-medium fa-3x social"></i></a>
+            <a href="https://anitab-org.github.io/mentorship-backend" rel="noopener noreferrer" target="_blank"><i id="fa fa-github" class="fa fa-github fa-3x social"></i></a>
```
Comment (isabelcosta) [inline docs/docusaurus.config.js]: Link to GitHub here would be https://github.com/anitab-org/mentorship-backend right :)

Comment (isabelcosta) [reply docs/docusaurus.config.js]: @sharonibose did you have the chance to look at this?

Comment (sharonibose) [reply docs/docusaurus.config.js]: Yeah, sorry for the delay. I had some technical issues in my area. Making the required changes now.

Comment (isabelcosta) [reply docs/docusaurus.config.js]: No worries. Glad you could come back :)

**Review by isabelcosta — APPROVED** [review]
Great work @sharonibose ! thank you for addressing the change requests 🤗

**Comment by sharonibose** [comment]
> Great work @sharonibose ! thank you for addressing the change requests 🤗

Thank you, @isabelcosta. I'm delighted to contribute.

**Review by vj-codes — APPROVED** [review]
LGTM 🎉

**Comment by vj-codes** [comment]
Adding label ready to merge since the gh pages action is passing.

**Pull request merged**